docs/components/networkmanager.md

```diff
@@ -94,6 +94,7 @@ If you are using Unity Relay to handle connections, however, **don't use `SetCon
 
 ## Disconnecting and Shutting Down
 
+### Disconnecting
 Disconnecting is rather simple, but you can't use/access any subsystems (that is, `NetworkSceneManager`) once the `NetworkManager` is stopped because they will no longer be available.  For client, host, or server modes, you only need to call the `NetworkManager.Shutdown` method as it will disconnect while shutting down.  
```

Comment:
```suggestion
When you disconnect from a network you can't use or access any subsystems, for example `NetworkSceneManager`, because they become unavailable when `NetworkManager` stops. For client, host, or server modes, call the `NetworkManager.Shutdown` method to disconnect and shut down at the same time.  
```